.github/workflows/tests.yaml

```diff
@@ -0,0 +1,47 @@
+name: Tests
```

Comment:
adding this to run tests automatically using Github actions. seems useful, but I can remove it if you rather not add it.